alb/main.tf

```diff
@@ -0,0 +1,84 @@
+locals {
+  lb_name = "${var.environment}-${var.name}"
```

Comment:
ebbe nem akarok most itt belekotni, es nem is neztem meg h tortent-e valtozas a tobbi module-nal h ez konzisztens legyen, de vegulis az is lehet majd h a dolgoknak a nevet nem keszitjuk elo igy local a module-ban, hanem csinalhatunk egy "data" module-t ami letrehozza a nevet/neveket es bepasszola...

Reply:
itt mi az environment name? (a korabbi terminologia szerint az h `testing`) azert kedezem mert eddig postfixes volt a naming (pl. aws acc `vulkaza-testing`) es sztem erdemes is lenne azt megtartani (domain-nel fordul csak de ott muszaj `testing.env.vulkaza.com`) ha nincs vmi kulonos oka a forditasnak
(lehet h itt azt adod be environment-nek h `vulkaza-testing` es nem valtozik a sorrend de akkor viszont nem biztos h konzisztens maganak az inputnak a neve)

Reply:
Ok, kiveszem. Mashol a roleokat nem igy hoztuk letre? env-rolename?

Reply:
egyelore csak felvetni szerettem volna mint kerdes h mit is csinalunk meg kellene...

...amugy a role es group neveknel az volt az ideologiam asszem h (vulkaza-)testing-lambda-hello es (vulkaza)-production-admins, szoval (vulkaza-)testing- csak a vulkaza- nincs kiirva...

azert nem koherens mert valoban vannak olyan resource nevek, amiknek indokoltabbnak tarottam az egeszet vegig kiirni (s3 bucket-nel pl eleg hasznos meg olyanoknal ahol neha jo tudni "kivulrol" nev, url, arn stb alapjan h melyik env-be tartozik vmi)

Reply:
a lb-nek szerintem is jo es kell a nevebe az env, de lehet a vulkaza- reszt is (project_name) beraktam volna ... es itt jon be h jo otlet volt-e ezekbe a module-okba tenni ezt az elnevezesi logikat, akkor meg jo otletnek tunt, de amugy lehet h felesleges bonyodalom lenne mashova tenni, plane most

Reply:
en nem tennem bele a project nevet, mert az csak nekunk fontos, maganak a cegnek/cliensnek nem, ok tudjak, h oket vulkazanak hivjak

s3-nal mas, ott akar azt is jelentheti, h cdn nelkul ki tudod e tenni publicba pl a statik asset bucketet, akkor a domainnek es a bucket nevnek egyezni kell

en s3-at domain-el nevezem el, pl `static.vulkaza.com` v `static.staging-vulkaza.com`  
mivel bucket nevek globally uniquenak kell lennie, igy a domain segit namespacingben is

Reply:
ok, de akkor majd le kell vadaszni h irtam e vhova project nevet ahova ez alapjan nem kene

ja, s3 global (de azert van amirol sejtheto h kb sosem akarod vagy kifejezetten sosem szeretne senki h public legyen[lasd alabb])

bucket neveknel mar most van inkoherencia btw.
vulkaza-testing-serverless-deployment
terraform-state.vulkaza-testing

Reply:
egyebkent egy cegnek is lehet tobb service-e, student.com es adultsxxx.com vagy vmi :D es akkor mar nem olyan biztos h akkor most melyik instance-t nezed epp

Reply:
lehet, ez kizarja a bucket elnevezest? azt azert csak tudod, h hova tarotzik egy bucket
de mind1 ez most, nekem jo az elso naming is, jo a masodik is, az biztos, h cdn nelkul ha http-t akarsz direktbe s3-rol akkor a domain nevnek es bucket nevnek egyezni kell

bar azt hiszem https-el nem megy, es az ugyis mar mindenhol szukseges

Reply:
az elso sort nem ertem, de majd beszeljuk meg mashol